docs/preview/02-Features/sinks/azure-application-insights.md

```diff
@@ -153,7 +153,7 @@ var host = Host.CreateDefaultBuilder()
 
                     string instrumentationKey = context.Configuration["ApplicationInsights:InstrumentationKey"];
 
-                    if (!String.IsNullOrWhiteSpace(instrumentationKey))
+                    if (!string.IsNullOrWhiteSpace(instrumentationKey))
```

Comment:
Changed this to lower-case, bc it's also the way we do it in Arcus, and I think it's also recommended by Microsoft.

Reply:
Idk what is 'recommended'.  Anyway, there should be no difference as `String` refers to `System.String` and `string` is just an alias for it.
cfr `long` -> `System.Int64` , `int` -> `System.Int32` etc...